### PR TITLE
Newsletter ipad issue

### PR DESCRIPTION
**Pull request by Kush2507**
### Fixed Issues (if relevant)

1. magento/magento2#21592:Newsletter subscription input box in footer: on iPad (9.7"), in portrait mode, placeholder text is not visible

### Manual testing scenarios (*)

1. Open any Luma storefront page on iPad (9.7")
2. Scroll down to newsletter subscription block in the global footer.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=23483) <br/>All committers have signed the CLA.

File: app/code/Magento/Sales/view/adminhtml/templates/order/address/form.phtml [hunk 1]
```diff
@@ -10,12 +10,12 @@
     </div>
 </div>
 
-<fieldset class="fieldset admin__fieldset-wrapper">
+<fieldset class="fieldset admin__fieldset-wrapper admin__fieldset">
```
Comment (sivaschenko) [inline app/code/Magento/Sales/view/adminhtml/templates/order/address/form.phtml]: Looks like these changes are for https://github.com/magento/magento2/pull/23482 can you please remove them from the pull request and leave only code changes related to this particular PR

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @Kush2507 , thanks for the pull request. Please see my review comments

File: app/code/Magento/Sales/view/adminhtml/templates/order/address/form.phtml (hunk 1 above)
Comment (Kush2507) [reply app/code/Magento/Sales/view/adminhtml/templates/order/address/form.phtml]: Hi @sivaschenko ,
I had created the branch from #23482 from there so file changes come with this branch also now I have removed the changes. I have resolved.

**Comment by sivaschenko** [comment]
@Kush2507 this PR and https://github.com/magento/magento2/pull/23482 still have conflicting (duplicate changes) please make sure all such changes are included only in one of the PRs

**Comment by sivaschenko** [comment]
@Kush2507 do you have any updates?

File: app/code/Magento/Sales/view/adminhtml/templates/order/address/form.phtml [hunk 2]
```diff
@@ -15,7 +15,7 @@
         <span><?= $block->escapeHtml($block->getHeaderText()) ?></span>
     </legend>
     <br>
-    <div class="form-inline" data-mage-init='{"Magento_Sales/order/edit/address/form":{}}'>
```
Comment (phoenix128) [inline app/code/Magento/Sales/view/adminhtml/templates/order/address/form.phtml:18]: Why do you need to change this phtml file in the scope of your PR?
Is it correct or is it an unwanted git inclusion?

**Review by phoenix128 — COMMENTED** [review]
Hello @Kush2507 ,
would you please check my comments?

**Comment by sidolov** [comment]
@Kush2507 , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**